packages/lesswrong/lib/modules/fragments.js

```diff
@@ -155,6 +158,7 @@ registerFragment(`
     canonicalBookId
     bannedUserIds
     user {
+      groups
```

Comment:
I have 40% or so on you overwriting the `...UsersMinimumInfo` here. Probably want to check that.